playbooks/labkeeper-allinone-deployment-test/run.yaml

```diff
@@ -0,0 +1,42 @@
+- hosts: all
+  become: yes
+  tasks:
+    - name: install required packages
+      shell: |
+        apt update -y
+        apt install python python-pip python3 python3-pip kpartx qemu-utils curl python-yaml \
+            debootstrap libffi-dev libssl-dev -y
+        pip install ansible
+      args:
+        executable: /bin/bash
+
+    - name: Prepare ubuntu user for running labkeeper
```

Comment:
Could we use become_user to running labkeeper?

Reply:
sure, oh, you mean the ubuntu user ? it  need user with ssh key